File: Test/app/src/main/java/com/example/test/RetroClient.java
package com.example.test;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetroClient {
    private static Retrofit getRetrofitInstance() {
        return new Retrofit.Builder()
                .baseUrl("https://test.kode-t.ru/")
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }

    public static ApiService getApiService() {
        return getRetrofitInstance().create(ApiService.class);
    }
}

File: Recipe/app/src/main/java/com/example/recipe/RetroClient.kt
package com.example.recipe

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory


class RetroClient {
    private fun getRetrofitInstance(): Retrofit {
        return Retrofit.Builder()
            .baseUrl("https://test.kode-t.ru/")
            .addConverterFactory(GsonConverterFactory.create())
            .build()
    }
    fun getApiService(): ApiService {
        return getRetrofitInstance().create(ApiService::class.java)
    }
}